CHANGELOG.md

```diff
@@ -4,10 +4,34 @@
 
 The full list of changes can be found in the compare view for the respective release at <https://github.com/open-telemetry/opentelemetry-proto/releases>.
 
-### Added
+## 1.8.0 - 2025-09-02
```

Comment:
I think this SHOULD NOT be merged sooner than on Tuesday (after Spec SIG meeting).

Reply:
Any reason to wait? The just adds factual information to CHANGELOG.md.